src/test/java/services/Service.java

```diff
@@ -8,5 +8,18 @@
 // (авторизация отдельно, операции с заказами отдельно,
 // регистрация пользователя отдельно и т.п.)
 
+import io.restassured.response.ExtractableResponse;
+import static io.restassured.RestAssured.given;
+import static requests.Request.requestSpec;
+
 public class Service {
+
+    public static ExtractableResponse getObj(String http){
```

Comment:
If I`m not mistaken, this approach looks like a common builder, maybe we can use a builder class instead of creating the object within a service class?